thread/src/main/kotlin/io/github/paulgriffith/kindling/thread/MultiThreadView.kt

```diff
@@ -453,7 +455,13 @@ object MultiThreadViewer : MultiTool {
     override fun open(paths: List<Path>): ToolPanel {
         return MultiThreadView(paths.sorted())
     }
-    //TODO: Implement Clipboard tool
+    override fun open(data: String): ToolPanel {
+        val tempFile = Files.createTempFile("kindling", "cb")
+        data.byteInputStream().use { threadDump ->
+            tempFile.outputStream().use(threadDump::copyTo)
+        }
+        return open(tempFile)
+    }
```

Comment:
I don't like extra temp files. This is why I'd rather have an abstraction over Paths/Files so we don't have to do this kind of thing.